// @kkiapay-org/widget-interfaces/dist/lib.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.PaymentProcess = exports.PaymentMethod = exports.TRANSACTION_STATUS = exports.EnvMode = void 0;
var EnvMode;
(function (EnvMode) {
    EnvMode["LIVE"] = "LIVE";
    EnvMode["SANDBOX"] = "SANDBOX";
})(EnvMode = exports.EnvMode || (exports.EnvMode = {}));
var TRANSACTION_STATUS;
(function (TRANSACTION_STATUS) {
    TRANSACTION_STATUS["PENDING"] = "PENDING";
    TRANSACTION_STATUS["SUCCESS"] = "SUCCESS";
    TRANSACTION_STATUS["FAILED"] = "FAILED";
    TRANSACTION_STATUS["INIT"] = "INIT";
})(TRANSACTION_STATUS = exports.TRANSACTION_STATUS || (exports.TRANSACTION_STATUS = {}));
var PaymentMethod;
(function (PaymentMethod) {
    PaymentMethod["momo"] = "momo";
    PaymentMethod["card"] = "card";
    PaymentMethod["direct_debit"] = "direct_debit";
})(PaymentMethod = exports.PaymentMethod || (exports.PaymentMethod = {}));
var PaymentProcess;
(function (PaymentProcess) {
    PaymentProcess["code_pin"] = "code_pin";
    PaymentProcess["user_information"] = "user_information";
    PaymentProcess["need_user_pin"] = "need_user_pin";
    PaymentProcess["need_otp_verification"] = "need_otp_verification";
    PaymentProcess["need_billing_details"] = "need_billing_details";
})(PaymentProcess = exports.PaymentProcess || (exports.PaymentProcess = {}));
